src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -295,8 +295,13 @@ export class SkyTabsetComponent
   public ngOnDestroy(): void {
     this.ngUnsubscribe.next();
     this.ngUnsubscribe.complete();
-    /*tslint:disable-next-line:no-null-keyword*/
-    this.setPathParamPermalinkValue(null);
+
+    const params = this.getPathParams();
+
+    if (Object.keys(params).length > 0) {
+      /*tslint:disable-next-line:no-null-keyword*/
+      this.setPathParamPermalinkValue(null);
+    }
```

Comment:
Why are these changes part of this PR?

Reply:
I uncovered an issue where errors were thrown while destroying a tabset component during unit tests unless a bunch of things were mocked out. This fix keeps a consumer's unit tests simple without having to mock things they don't care about.